src/windows/CameraProxy.js

```diff
@@ -758,16 +759,31 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
         }, successCallback, errorCallback);
     };
 
-    // add and delete focus eventHandler to capture the focus back from cameraUI to app 
-    window.addEventListener("focus", savePhotoOnFocus);
-    cameraCaptureUI.captureFileAsync(WMCapture.CameraCaptureUIMode.photo).done(function(picture) {
+    // if windows phone 10, add and delete focus eventHandler to capture the focus back from cameraUI to app
+    if (navigator.appVersion.indexOf('Windows Phone 10.0') >= 0) { 
```

Comment:
I think this is our best option. Also the other checks in this file seems to use this property as well. It would break when the version changes though yes. 
Non phone windows 10 works fine without the focus handler.